migrations/20200606105847-convert-collective-long-description-to-html.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import Promise from 'bluebird';
+import showdown from 'showdown';
+
+const converter = new showdown.Converter();
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const collectives = await queryInterface.sequelize.query(
+      `
+        SELECT * FROM "Collectives"
+        WHERE LENGTH("longDescription") > 0 AND "longDescription" NOT LIKE '<%';
+      `,
```

Comment:
No need to include deleted collectives

```suggestion
        WHERE LENGTH("longDescription") > 0 AND "longDescription" NOT LIKE '<%'
        AND "deletedAt" IS NULL;
      `,
```